UnityProject/Assets/Scripts/Weapons/Gun.cs

```diff
@@ -652,6 +651,13 @@ private void DequeueAndProcessServerShot()
 			{
 				b.Suicide(shooter, this, damageZone);
 			}
+			else if (ProjectilesFired != 1)
```

Comment:
You can just have one `else` statement without `if`. Iterate through one projectile.